platform-sdk/docs/proposals/template.md

```diff
@@ -0,0 +1,135 @@
+# Proposal Title Goes Here
+
+---
+
+## Summary
+
+Give a 1-3 sentence summary of the capability in the design proposal.
+
+| Metadata           | Entities                            | 
+|--------------------|-------------------------------------|
+| Designers          | John, Mary, alpha-team              |
+| Functional Impacts | Services, DevOps, Mirror Node, etc. |
+| Related Proposals  | Proposal-1, Proposal-2              |
+| HIPS               | HIP-1, HIP-2,                       |
```

Comment:
are the given entries just examples or do you assume that each proposal should have filled out that keys? I assume it makes sense to add a short note to make that clear